subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -56,6 +56,13 @@ config BT_L2CAP_DYNAMIC_CHANNEL
 	  This option enables support for LE Connection oriented Channels,
 	  allowing the creation of dynamic L2CAP Channels.
 
+config BT_L2CAP_DYNAMIC_CHANNEL_ECRED
+	bool "L2CAP Enhanced Credit Based Flow Control support"
+	depends on BT_L2CAP_DYNAMIC_CHANNEL
```

Comment:
I would default to y though since ECRED other than the new L2CAP request opcodes it is basically the same as COC, in the other hand the choice of using L2CAP_CREDIT_BASED_CONNECTION_REQ by PTS really beats me, it does looks like more of a PTS doing + 1 to known opcodes and end up failing so every new opcode introduced would break the existing test.

Reply:
I prefer to have new features as default off. If applications need the feature they can enable it.

Applications moving between versions of zephyr would otherwise have to go and find all new features and disable them to avoid increasing the RAM and Flash usage unnecessarily.

Reply:
Agreed, we should not enable this by default, this is a new, relatively unusual feature.